pkgs/development/tools/continuous-integration/github-runner/default.nix

```
@@ -279,10 +285,13 @@ stdenv.mkDerivation rec {
     wrap Runner.Listener
     wrap Runner.PluginHost
     wrap Runner.Worker
-    wrap run.sh
-    wrap env.sh
+    wrap run.sh --run 'export RUNNER_ROOT=''${RUNNER_ROOT:-$HOME/.github-runner}'
+    wrap env.sh --run 'cd $RUNNER_ROOT'
```

Couldn't you also use `--chdir` here?

```suggestion
    wrap env.sh --chdir "$RUNNER_ROOT"
```